blog/nse_portfolio_sorts/index.qmd

```diff
@@ -0,0 +1,931 @@
+---
+title: "Non-standard errors in portfolio sorts"
+author: "Patrick Weiss"
+date: "2023-05-10"
+description: An all-in-one implementation of non-standard errors in portfolio sorts.
+image: thumbnail.jpg
+categories: 
+  - Replications
+---
+
+# What does this post cover?
+
+Personally, I find non-standard errors[^1] very exciting and an important angle on academic research activity on a general level. Generally, non-standard errors are induced by methodological choices, which adds another layer of uncertainty to reported estimates. Luckily, Stefan Voigt asked me to join forces contributing to Menkveld et al. (2023). Furthermore, if you look back at the history of Tidy Finance, we included a chapter on [Size sorts and p-hacking](https://www.tidy-finance.org/size-sorts-and-p-hacking.html) early, starting an assessment of different choices in portfolio sorts. Recently, my co-authors (shout out to [Dominik Walter](https://sites.google.com/view/dominikwalter/startseite) and [Rüdiger Weber](https://sites.google.com/site/ruedigercweber/)) and I uploaded an update to our working paper on the topic. [Non-Standard Errors in Portfolio Sorts](http://dx.doi.org/10.2139/ssrn.4164117)[^2] (WWW below) thoroughly studies how methodological choices influence return differentials estimated from portfolio sorts. One of the conclusions is that we should embrace non-standard errors and report distributions of return differentials.
+
+This blog article will teach you how to do portfolio sorts with non-standard errors in mind, i.e., vary over possible decisions to gain a deeper insight into return differentials. We will estimate one premium's distribution instead of a single return differential. Tidy Finance with R is the inspiration for the code alongside the replication code for WWW in this [Github repository](https://github.com/patrick-weiss/PortfolioSorts_NSE). Ultimately, you will know how to sort portfolios on the variable *asset growth* in nearly 70,000 ways.
+
+This blog post is long, albeit not overly complicated. If you are new to `R` or portfolio sorts, I suggest reading the Chapter [Size sorts and p-hacking](https://www.tidy-finance.org/size-sorts-and-p-hacking.html) before jumping into this post. Unfortunately, I have to skip many interesting points on the implementation and the economic background (read WWW for this part). Otherwise, this blog would be far too long. If there is a need to expand on individual aspects, let me know. 
+
```

Comment:
Just for fun, hab mal GPT-4 die ersten 3 Absätze neu schreiben lassen :)

Welcome to our latest blog post where we delve into the intriguing world of non-standard errors[^1], a crucial aspect of financial academic research. These non-standard errors often stem from the methodological decisions we make, adding an extra dimension of uncertainty to the estimates we report. I had the pleasure of working with Stefan Voigt on a contribution to Menkveld et al. (2023), which was an exciting opportunity.

Our journey with Tidy Finance has always been focused on promoting reproducibility in finance research. We began this endeavor by introducing a chapter on [Size sorts and p-hacking](https://www.tidy-finance.org/size-sorts-and-p-hacking.html), which initiated a comprehensive analysis of portfolio sorting choices. Recently, my fellow authors, [Dominik Walter](https://sites.google.com/view/dominikwalter/startseite) and [Rüdiger Weber](https://sites.google.com/site/ruedigercweber/), and I introduced an update to our working paper, [Non-Standard Errors in Portfolio Sorts](http://dx.doi.org/10.2139/ssrn.4164117)[^2]. This paper delves into the impact of methodological choices on return differentials derived from portfolio sorts. Our conclusion? We need to accept non-standard errors and report the entire range of return differentials.

This blog post will guide you through conducting portfolio sorts while keeping non-standard errors in perspective. We'll explore the multitude of possible decisions that can impact return differentials, allowing us to estimate a premium's distribution rather than a single return differential. The code we'll be using is inspired by Tidy Finance with R and the replication code for the WWW paper, which can be found in this [Github repository](https://github.com/patrick-weiss/PortfolioSorts_NSE). By the end of this post, you'll have the knowledge to sort portfolios based on asset growth in nearly 70,000 different ways.

While this post is detailed, it isn't overly complex. However, if you're new to R or portfolio sorting, I recommend first checking out our chapter on [Size sorts and p-hacking](https://www.tidy-finance.org/size-sorts-and-p-hacking.html). Due to the length constraints, we'll be skipping over certain details related to implementation and the economic background (you can find these in the WWW paper). If there are particular aspects you'd like to delve into further, please feel free to reach out.

Reply:
I quite like this. Implemented some of the changes in Commit https://github.com/tidy-finance/website/pull/34/commits/a1850e2b8819856a37184cac34149c2cdb841e56.